src/20_Components/amp-form.html

```diff
@@ -106,5 +111,46 @@
         </template>
     </div>
   </form>
+  <!-- #### Other form input samples -->
```

Comment:
Can you also add samples for:

datetime-local
email
hidden
month
number
radio
range
tel
time
url
week
reset
search

I would put all inputs into a single form (also the checkbox one).

Also: What about image submit buttons? Are they allowed?